src/appleseed.shaders/src/max/as_max_blend_material.osl

```diff
@@ -68,23 +68,23 @@ surface as_max_blend_material
     ClosureOut = BaseMtl;
     
     if (LayerMtl_1)
-        ClosureOut = mix(ClosureOut, LayerMtl_1, (MixColor_1 * MaskColor_1));
+        ClosureOut = mix(ClosureOut, LayerMtl_1, (MixAmount_1 * MaskColor_1));
```

Comment:
I suggest removing the mask colors if they are never used.

Reply:
Mask color is the texture, we use them. If no texture then it's 1 so only mix amount affects the mix.

Reply:
Oh ok, sorry.